docs/getting-started.md

```diff
@@ -0,0 +1,71 @@
+# Getting started
+
+You will need to be running the version of NodeJS set in the `.nvmrc` file in the root of the repository and `Yarn` package manager.
+
+You can quickly install Yarn by running `npm i -g yarn`. If you are running multiple versions of Node on your machine you will need to make sure you install Yarn on the specific version Node you are using as the installation will not jump between different versions of NodeJS.
+
+## Installation
+
+Once you have cloned the repo, run `npm ci` to install the required dependencies.
+
+Next you will need to install the Form Runner. You can do that by running `npm run form-runner:install`. The form runner will be cloned into the `lib` folder - this folder should never be committed into the repo.
+
+## Config
+
+You will need to set up your `.env` file to contain the correct values. Below is an empty example that you will need to fill in.
+
```

Comment:
we've put them in Keybase now. I think documenting the values as a .env file with defaults would be good, but we can also setup the code to just read the env file from Keybase (it sets up a volume in the exact same place for all machines)